src/mem/cache/cache_probe_arg.hh

```diff
@@ -84,17 +84,23 @@ struct CacheDataUpdateProbeArg
     Addr addr;
     /** Whether the block belongs to the secure address space. */
     bool isSecure;
+    /** Block original requestor */
+    const RequestorID requestorID;
```

Comment:
Feel free to say "no", but would it make sense to store the thread context here instead of in the probe?

Reply:
This should be ID of either the cache object or the prefetcher object that brought in the line. I'm not sure how we could get this from the thread context here.

Reply:
Ack. Thanks 😄